frontend/src/components/auction/AuctionList.tsx

```
@@ -0,0 +1,63 @@
+import React from 'react';
+import AuctionCard from './AuctionCard';
+import logo from '../../assets/bidgood_logo.png';
+
+interface AuctionListProps {
+    
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)